.github/workflows/cloud_csharp_client_test.yaml

```diff
@@ -0,0 +1,100 @@
+name: Hazelcast cloud test with csharp client
+
+on:
+  workflow_dispatch:
+    inputs:
+      organization_name:
+        description: Default is hazelcast, but if you would like to run the workflow with your forked repo, set your github username
+        required: true
+        default: hazelcast
+      branch_name:
+        description: Name of the branch to test client from
+        required: true
+        default: master
+      base_url:
+        description: Base url of the cloud env. I.e https://uat.hazelcast.cloud
+        required: true
+        default: https://uat.hazelcast.cloud
+      hzVersion:
+        description: Version of hazelcast (For cloud standard cluster, it has different versions, i.e 5.0.2-2)
+        required: true
+
+jobs:
+  test_standard_cluster:
+    runs-on: ubuntu-latest
+    name: Hazelcast cloud standard cluster tests with csharp
+    steps:
+      - name: Checkout to scripts
+        uses: actions/checkout@v2
+        
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+          
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: |
+            3.1.x
+            5.0.x
+            6.0.x
+          
+      - name: Checkout to ${{ github.event.inputs.branch_name }}
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.inputs.organization_name }}/hazelcast-csharp-client
+          path: client
+          ref: ${{ github.event.inputs.branch_name }}
+        
+      - name: Copy reqired files to under test folders
+        run: |
+          rm -rf $GITHUB_WORKSPACE/client/src/Hazelcast.Net.Testing/Remote
+          cp -R $GITHUB_WORKSPACE/HazelcastCloudTests/csharphazelcastcloudtests/Remote $GITHUB_WORKSPACE/client/src/Hazelcast.Net.Testing/Remote/
+          cp -R $GITHUB_WORKSPACE/HazelcastCloudTests/csharphazelcastcloudtests/CloudTests $GITHUB_WORKSPACE/client/src/CloudTests/
+          ls $GITHUB_WORKSPACE/client/src/CloudTests
```

Comment:
is this needed?

Reply:
I'm not sure. We may get rid of that since each time everything re-built.  The whole structure is created by Ali. I moved the structure and tests from his repo to Hazelcast with some touch. I think as a QA guy, his way of doing things. I can reconstruct everything but needs planning.

Reply:
I remember now. He collects all projects into a directory which is aligned to work together with its references.